.gitmessage

```diff
@@ -0,0 +1,10 @@
+[component(s)] Title
+
+Description
+   
+path/file1:
+  Specific changes in file1.
+	   
+# path/file2:
+#   Specific changes in file2.
+# ...
```

Comment:
Could you add to the end of this file, a commented out section that explains some things, like what the suitable components are, that you can use wildcards to show (to a human) sets of files.  And that the title is not a full sentence and also doesn't need to end in a period (a surprising grammar thing, am I being pedantic?)